sdk/core/core-util/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### Features Added
 
 - Add helper type guards `isDefined`, `isObjectWithProperties`, `objectHasProperty`.
+- Update `delay` method to handle `abortSignal` functionality
```

Comment:
I merged a PR yesterday to bump core-util version to 1.1.0.  This PR should probably wait until after we release core-util today.